framework/db/ConnectionInterface.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\db;
+
+use yii\base\NotSupportedException;
+
+/**
+ * ConnectionInterface defines a connection to a database.
+ *
+ * @author Nelson J Morais <njmorais@gmail.com>
+ * @package yii\db
+ */
```

Comment:
Need `@since` as the interface is new.

Reply:
done! used 2.0.15 since that's the version on #14997 issue.